.github/workflows/build-wheels.yml

```diff
@@ -13,8 +13,53 @@ on:
       - '*wheel*'  # must quote since "*" is a YAML reserved character; we want a string
 
 jobs:
+  generate_backwards_compatibility_data:
+    name: Generate Backwards Compatibility Data
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      
+      # Based on https://github.com/TileDB-Inc/conda-forge-nightly-controller/blob/51519a0f8340b32cf737fcb59b76c6a91c42dc47/.github/workflows/activity.yml#L19C10-L19C10
+      - name: Setup git
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "runneradmin@users.noreply.github.com"
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+
+      - name: Print Python version
+        run: |
+          which python
+          which pip
+          python --version
+
+      - name: Build Indexes
+        run: |
+          # Get the release tag.
+          release_tag=$(git describe --tags --abbrev=0)
+          echo $release_tag
+
+          # Install dependencies.
+          cd apis/python && pip install . && cd ../..
+
+          # Generate data.
+          python backwards-compatibility-data/generate_data.py $release_tag
+
+          # Push this data to main.
+          git fetch origin main
+          git checkout main
+          git add backwards-compatibility-data/data/
+          git commit -m "[automated] Update backwards-compatibility-data for release $release_tag"
+          git push origin main
```

Comment:
Should we push directly to main without review?

Reply:
I don't feel too strongly, but I suppose it would be safer if we opened up a PR that we could then approve. I can change to that.

Reply:
Okay updated. [Here is the job](https://github.com/TileDB-Inc/TileDB-Vector-Search/actions/runs/7609637929/job/20721285388) and this is the resulting PR: #208. Note that there are lots of extra changes in that PR, but once the code in this PR gets checked in, that PR will just be the new data.

Reply:
Thanks!